tests/test_middleware.py

```diff
@@ -2,7 +2,7 @@
 from django.http import HttpResponse
 
 import mock
-from mohawk.exc import MacMismatch
+from mohawk.exc import MisComputedContentHash
```

Comment:
This is for a mohawk change, right? I don't see the mohawk version updated anywhere. Were you going to pin all dependency versions in [requirements/dev.txt](https://github.com/kumar303/hawkrest/blob/master/requirements/dev.txt) or is that no longer necessary now that older Djangos are removed?